packages/cli/generation/ir-generator/src/filtered-ir/IrGraph.ts

```diff
@@ -61,9 +60,6 @@ export class IrGraph {
         if (this.hasAudience(audiences)) {
             this.typesNeededForAudience.add(typeId);
             this.addSubpackages(declaredTypeName.fernFilepath);
-            this.types[typeId]?.referencedSubpackages.forEach((fernFilePath) => {
-                this.addSubpackages(fernFilePath);
-            });
```

Comment:
note we should still keep this logic, `referencedSubpackages` is different from `referencedTypes`